syncer/sync.py

```diff
@@ -22,52 +17,63 @@
 print("Done!")
 
 print("Connecting to Pachyderm")
-pc = pach.Client(host=os.environ["PACHD_SERVICE_HOST"], port=os.environ["PACHD_SERVICE_PORT"])
+PACHY_CLIENT = pach.Client(
```

Comment:
Very minor nit, but in the past we've gotten the feedback to prefer "pach" over "pachy", which close to a particular British slur.